app/controllers/attendance_committee_controller.rb

```diff
@@ -1,47 +1,45 @@
 class AttendanceCommitteeController < ApplicationController
+    before_action :confirm_logged_in, only: [:index, :show]
+    before_action only: [:index] do
+      confirm_logged_in('Please Log In')
+    end
+    before_action :confirm_staff, only: [:index, :show]
+
     class CommitteePoints
         attr_accessor :committee, :social_points, :fundraising_points, :campus_relations_points, :pr_points, :community_outreach_points, :give_back_points
       end
 
     def index
```

Comment:
Add comments before defs saying what this route accomplishes

Reply:
Change with 3e30aff